basics/form.php

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible">
+    <title>PHP Form</title>
+</head>
+
+<body>
+
+<form action="form.php" method="post" enctype="multipart/form-data">
+    <label for="first_name">First Name:</label>
+    <input type="text" id="first_name" name="first_name" pattern="[A-Za-z]+" required>
+
+    <br><br>
+
+    <label for="last_name">Last Name:</label>
+    <input type="text" id="last_name" name="last_name" pattern="[A-Za-z]+" required>
+
+    <br><br>
+    <label for="image">Select Image:</label>
+    <input type="file" name="image" id="image" accept="image/*">
+
+    <br><br>
+
+    <label for="full_name">Full Name:</label>
+    <input type="text" id="full_name" name="full_name" disabled value="<?php
+    echo isset($_POST['first_name']) && isset($_POST['last_name']) ? $_POST['first_name'] . ' ' . $_POST['last_name'] : '';
```

Comment:
Full Name field's functionality should be while you writing anything in first name & last name field at that time the full name field should be filled with those fields values. But here it will show the value when you submit the form.